app/services/db/service_provider_quota_limit/notify_if_any_sp_over_quota_limit.rb

```diff
@@ -5,7 +5,9 @@ def self.call
         return unless Db::ServiceProviderQuotaLimit::AnySpOverQuotaLimit.call
         email_list = IdentityConfig.store.sps_over_quota_limit_notify_email_list
         email_list.each do |email|
-          ReportMailer.sps_over_quota_limit(email).deliver_now_or_later
+          # rubocop:disable IdentityIdp/MailLaterLinter
+          ReportMailer.sps_over_quota_limit(email).deliver_now
+          # rubocop:enable IdentityIdp/MailLaterLinter
```

Comment:
🤔 what if we only applied the lint to `UserMailer` and its subclasses instead of `ApplicationMailer`?

Reply:
I think that'd make sense, I don't know how to that off-hand from a brief look at the current linter.